<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_Password</class>
 <widget class="QDialog" name="Show_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy password to clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy login to clipboard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
